build.gradle

```
@@ -4,19 +4,24 @@ plugins {
 
 group 'org.testcontainers.cloud.turbomode.sample'
 version '1.0-SNAPSHOT'
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter:5.9.0'
-    testImplementation platform('org.testcontainers:testcontainers-bom:1.17.4')
+    testImplementation platform('org.testcontainers:testcontainers-bom:1.17.5')
     testImplementation "org.testcontainers:testcontainers"
     testImplementation "org.testcontainers:junit-jupiter"
     testImplementation 'ch.qos.logback:logback-classic:1.3.3'
     testImplementation 'org.assertj:assertj-core:3.23.1'
-    testImplementation "org.testcontainers:kafka:1.17.4"
+    testImplementation "org.testcontainers:kafka"
+    testImplementation 'org.testcontainers:kafka'
+    implementation 'org.apache.kafka:kafka-clients:1.0.0'
+    testImplementation group: 'org.awaitility', name: 'awaitility', version: '3.1.6'
```

```suggestion
    testImplementation 'org.awaitility:awaitility:3.1.6'
```
Short-form everywhere